examples/2d/frustum_sprites.rs

```diff
@@ -0,0 +1,83 @@
+use bevy::{
+    diagnostic::{FrameTimeDiagnosticsPlugin, LogDiagnosticsPlugin},
+    prelude::*,
+};
+
+const CAMERA_SPEED: f32 = 1000.0;
+
+pub struct CameraTimer(Timer);
+pub struct Direction(Vec3);
+
+fn main() {
+    App::build()
+        .insert_resource(ReportExecutionOrderAmbiguities)
+        .add_plugin(LogDiagnosticsPlugin::default())
+        .add_plugin(FrameTimeDiagnosticsPlugin::default())
+        .insert_resource(bevy::log::LogSettings {
+            level: bevy::log::Level::DEBUG,
+            ..Default::default()
+        })
+        .add_plugins(DefaultPlugins)
+        .add_startup_system(setup.system())
+        .add_system(move_camera.system())
+        .run()
+}
+
+fn setup(
+    commands: &mut Commands,
+    assets: Res<AssetServer>,
+    mut materials: ResMut<Assets<ColorMaterial>>,
+) {
+    let tile_size = Vec2::new(64.0, 64.0);
+    let map_size = Vec2::new(320.0, 320.0);
+
+    let half_x = (map_size.x / 2.0) as i32;
+    let half_y = (map_size.y / 2.0) as i32;
+
+    let material = materials.add(assets.load("textures/icon.png").into());
```

Comment:
Maybe `sprite` would be a better name for this variable?

Reply:
Could you reuse "assets/branding/icon.png" here? This would avoid adding a new asset to the repository